views/pages/consent.njk

```diff
@@ -1,36 +1,171 @@
+                        'no': 'consent.no'
+                    },
+                    data.consent,
+                    'consent.question',
+                    errors,
+                    {
+                        'safe': true
+                    }
+                )
+            }}
+
+            <table class="md:ml-12 align-top text-lg">
+                <tr>
+                    <td class="align-top">
+                        <svg class="h-8 w-8 inline" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M19.511 17.98L10.604 1.348a.697.697 0 0 0-1.208 0L.49 17.98a.675.675 0 0 0 .005.68c.125.211.352.34.598.34h17.814a.694.694 0 0 0 .598-.34.677.677 0 0 0 .006-.68zM11 17H9v-2h2v2zm0-3.5H9V7h2v6.5z"></svg>
+                    </td>
+                    <td class="md:pl-2">
+                        {{  __('consent.i_understand_intro') }}
+                        <ul class="pl-5">
+                            <li>{{  __('consent.i_understand_1') }}</li>
+                            <li>{{  __('consent.i_understand_2') }}</li>
+                            <li>{{  __('consent.i_understand_3') }}</li>
+                        </ul>
+                    </td>
+                    <tr/>
+            </table>
+
+            <br/>
+
+            {{ radioButtons(
+                    'signature_mode',
+                    {
+                        'draw': 'consent.signature_draw',
+                        'type': 'consent.signature_type'
+                    },
+                    data.signature_mode if data.signature_mode else 'draw',
+                    'consent.signature_option',
+                    errors,
+                    {
+                        'required': true
+                    }
+                )
+            }}
+
+            <br/>
 
-        <p>{{ __('section2.delay_warning_preamble') }}</p>
-        <ul>
-            <li>{{ __('section2.delay_warning_1') }}</li>
-            <li>{{ __('section2.delay_warning_2') }}</li>
-            <li>{{ __('section2.delay_warning_3') }}</li>
-        </ul>
+            <div id="signature_draw" class="{{ 'has-error' if errors and errors['signature_draw_data'] }}">
+                <label>
+                    <span aria-hidden="true" class="required">*</span>
+                    {{ __('consent.signature_draw_instructions') }}
+                    <span class="required">{{ __('required') }}</span>
+                </label>
+                {% if errors and errors['signature_draw_data'] %}
+                    {{ validationMessage(errors['signature_draw_data'], 'signature_draw_data') }}
+                {% endif %}
+                <div id="signature_draw" class="relative w-3-4 border-2 border-black">
+                    <div id="signature_pad_clear" class="bottom-0 w-full text-right pr-4 pt-4 leading-none bg-white">
+                        <a onclick="$('#signature_pad').jSignature('clear')" class="no-underline text-gray-500">Clear</a>
```

Comment:
Sadly the 'Clear' text fails contrast checks. We'll have to use `text-gray-700`.